docker/devbox/nginx_wrapper.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+mkfifo /var/log/nginx/std.log
+nginx -g "daemon off;" > /var/log/nginx/std.log
```

Comment:
So this one of the things I've been struggling with. It seems that nginx docker is set up to write to /dev/stdout, so on nginx you have a symlink /var/log/nginx/access.log => /dev/stdout (error log is similar). The problem with that setup caused the exporter to throw a panic(when log writing stops for some reason the app tries to seek in that file, but you can't seek in pipe). This solution blocks the pipe, so the app waits for the pipe instead of seeking the new line after write is finished. The lib used to doing this is quite stable, and I couldn't get it to work the other way:/